### ISSUE TITLE
ORCID feed stopped working

### ISSUE THREAD

**Issue by frnz65** [ORCID-EU-Labs/orcid-feed#22]
Hi,

I can't make ORCID feed work since yesterday.  http://feed.labs.orcid-eu.org/ shows a page with the error
"Ruby (Rack) application could not be started", with a list of more errors below.

Francesco

**Comment by lizkrznarich** [comment]
Hi Francesco,
Sorry about that! Logs had filled up the disk space on that server, so we cleared them out and restarted. ORCID Feed should be back up and running.

**Comment by ulli-steiner** [comment]
Liz,

Thanks for fixing this so quickly!

Ulli

**Comment by TomDemeranville** [comment]
Hi @ulli-steiner and @frnz65 please let us know where your integration is.  We are trying to migrate feed users to using https://github.com/ORCID/orcid-js instead as feeds is not part of or core service and we'd love to help you move over.

**Comment by ulli-steiner** [comment]
Hi,

While I recognise the I like power of orcid-js,  the feed is much simpler to use.  I don’t have much time for debugging code and I could not get the “example.html” to work off the bat.

Firefox returns the error:

SyntaxError: expected expression, got keyword 'default'         xmle4x.js:11:4

While Safari comes up with a bunch of errors.  Either the github documentation is insufficient or there is a bug.  Whichever, I am prepared only to start implementing this if the published ample code works.

With best wishes,

Ulli

**Comment by TomDemeranville** [comment]
Hi @ulli-steiner 

The issue with chrome is documented in the README.md and does not affect the execution of the script.  It is a warning issued for the citeproc dependencies attempting to support legacy browsers.

Regarding Safari, I will test it in safari.  If you could point me towards your site then I'll have a look.

**Comment by ulli-steiner** [comment]
Tom,

It still does not work - It does not produce a publication list in either FireFox or Safari ….

Ulli

**Comment by ulli-steiner** [comment]
Tom,

To get a handle on this, I instead tried the tutorial on your web-page and was stymied already in first step.  The first block of code produces the following error:

TypeError: swagger.apis['Public API v2.0_rc1'] is undefined
window.swagger<.success()
 Test.html:9
[3]</SwaggerClient.prototype.buildFromSpec()
 swagger-client.js:638
[6]</Resolver.prototype.finish()
 swagger-client.js:1527
[6]</Resolver.prototype.resolve()
 swagger-client.js:1436
[3]</SwaggerClient.prototype.build/obj.on.response()
 swagger-client.js:402
[5]</SwaggerHttp.prototype.execute/responseInterceptor()
 swagger-client.js:893
[5]</SwaggerHttp.prototype.execute/obj.on.response()
 swagger-client.js:897
[5]</SuperagentHttpClient.prototype.execute/<()
 swagger-client.js:1113
[161]</Request.prototype.callback()
 swagger-client.js:17526
Request/<()
 swagger-client.js:17167
[16]</Emitter.prototype.emit()
 swagger-client.js:5917
[161]</Request.prototype.end/xhr.onreadystatechange()
 swagger-client.js:17608
 Test.html:9:7

Ulli

**Comment by TomDemeranville** [comment]
You are correct, my apologies.  There is an issue on the main ORCID
registry that is preventing the swagger client from working.  I am looking
at it now and will let you know when it's fixed.

**Comment by ulli-steiner** [comment]
Many thanks,

Ulli

**Comment by TomDemeranville** [comment]
Ok, I've submitted a patch to the ORCID registry and also a couple of bug fixes to ORCID-js.  I'll post again once the changes are live

**Comment by frnz65** [comment]
Hello,

thank you. The feed is quite simple to use, that’s why I started with that. Using the API with the OAuth authentication takes a lot more time to set up and I haven’t found enough help here so far. Happy to use what you suggest, I will try now,

thanks,

Francesco

**Comment by frnz65** [comment]
Just tried the example,  I get a bunch of errors in javascript with Chrome: 

swagger-client.js:1 Uncaught SyntaxError: Unexpected token <
xmle4x.js:11 Uncaught SyntaxError: Unexpected token default
orcid.js:38 Uncaught ReferenceError: SwaggerClient is not definedcreatePublicV2ORCIDClient @ orcid.js:38(anonymous function) @ example.html:15j @ jquery.min.js:2k.fireWith @ jquery.min.js:2n.extend.ready @ jquery.min.js:2I @ jquery.min.js:2

Francesco Arneodo
Associate Professor, Physics Program Head
NYU Abu Dhabi
Office Tel (UAE): +971 2 628 5331
Mobile (UAE): +971 56 689 9498

**Comment by ulli-steiner** [comment]
Still not working, as far as I can tell ….

Ulli

**Comment by TomDemeranville** [comment]
Hi both.  The code fix has been merged but I'm not sure when it will be live.  I'll keep you posted.

**Comment by frnz65** [comment]
thank you, it doesn’t seem to be live yet.

Francesco

**Comment by ulli-steiner** [comment]
Hi Tom

it is not really confidence inspiring if a fix takes such a long time.  If I integrate this into my web pages, will it take days or weeks to fix if something goes wrong?

I hope you keep the orcid feed alive still for a while …..

Best wishes,

Ulli

**Comment by TomDemeranville** [comment]
Hi Both,

Apologies again, I should have been clearer.  We have a roughly two week release cycle and the fix is part of wider changes.  It should be out soon.  ORCID-js is a very new development and not yet fully supported by integration tests in the core service.  This will happen soon, ensuring it never breaks in production.

ORCID feeds in not part of the core ORCID service.  It's on a third party server infrastructure, written in a different language and on a different server stack.  This makes it difficult for us to support and maintain and is one of the reasons we're depreciating it.

**Comment by ulli-steiner** [comment]
Thanks - I will start the migration as soon as orcid-js is stable….

Ulli

**Comment by ulli-steiner** [comment]
Dear Tom,

Your changes seem to have kicked in.  I got the example code to work.  Great stuff!  I am looking forward to implement it.

A few observations:
1. For a long publication list, this is very slow.  Does this mean the pub.orcid.org api is slower than feed.labs.orcid-eu.org, or is it the additional overhead (note that I have not installed the styles, but load them directly from "raw.githubusercontent.com/citation-style-language/styles/master/“)?
2. The code throws up the a few of these errors:

response-code":500,"developer-message":"org.jbibtex.TokenMgrError: Lexical error at line 1, column 4.  Encountered: \"\u00e4\" (228), after : \"\"","user-message":"Something went wrong in ORCID.","error-code":9008,"more-info":"http://members.orcid.org/api/api-error-codes”

It seems that my publication list has a few utf-8 encoded umlauts that the code does not seem to handle well.

For some of the styles, an exception halts the execution of the code:

TypeError: null is not an object (evaluating 'Item[variable].length’):  citeproc.js:9557.  My guess is that this effects the styles that list the title of the publication, which fails when there is a special character in the title.

Give it a try yourself:  my ORCID is:  0000-0001-5936-339X  I get a list with only a few errors for american-physics-society.csl, but failure example for nature.cls

Best wishes,

Ulli

**Comment by TomDemeranville** [comment]
@ulli-steiner thanks for the detailed update.  I'll find time next week and get the bugs ironed out.

The time could be that it now resolves each work metadata individually from Datacite or Crossref.  I think we could improve it to add works asynchronously to the page as they are loaded rather than waiting for the entire list to be generated.

Tom.

**Comment by ulli-steiner** [comment]
Tom,

The problem may in fact arise from the Crossref input into Orcid, which is very flaky in terms of non-ASCII characters

Ulli

**Comment by frnz65** [comment]
Hello,

where do I find the new version? it seems that here https://github.com/ORCID/orcid-js https://github.com/ORCID/orcid-js the last commit is still 17 days ago.

Thank you

Francesco

**Comment by TomDemeranville** [comment]
@frnz65 it was a change to the ORCID service that fixed it, not ORCID-JS.   It should be working fine now.

**Comment by ulli-steiner** [comment]
Working:  only kind of:  the code still produces a lot of errors.

Ulli

**Comment by TomDemeranville** [comment]
Would you mind raising the errors as issues on the ORCID-JS repo?  That way
I can get the addressed.

Cheers.

**Comment by ulli-steiner** [comment]
Tom,

interestingly, this depends on the computer from which I access the “example.html” (orcid-js is installed on our web-server).   Calling example.html from within my institute produces much fewer errors, than accessing it from home.

As mentioned before, the code hicks-up over utf-8 codes (Umlauts, accents).  In the best case this produces a "[Error] Failed to load resource: the server responded with a status of 500 (Internal Server Error)” when accessing the “pub.orcid.org” record, and the record is skipped.  If the utf-8 code appears in the publication title, the code aborts in citeproc.js: TypeError: null is not an object (evaluating 'Item[variable].length’)

If I call example.html from my home computer (outside the institute domain) the list of errors is much longer and more diverse, including errors when calling crossref, etc.  I’ll compile a list of these one of the coming evenings.

The bottom line is that for the ORCID records that I administer, the orcid-js package is not yet usable.  I am however very much interested in it and hope to be able to contribute to its improvement.

Best wishes,

Ulli

**Comment by ulli-steiner** [comment]
Tom,

further to my previous e-mail:  the error I get when accessing orcid-js from an outside computer is a bit curious:

Failed to load resource: the server responded with a status of 504 (Gateway Timeout).  This occurs when attempting to access data.crossref.org, but only if the computer that loads the page is not on the same network as the server.

Also, it seems that   data.crossref.org is sometime called with mis-formed DOIs:  http://data.crossref.org/10.1002%2Fadfm.201502392 instead of  http://data.crossref.org/10.1002/adfm.201502392

Best wishes,

Ulli

> Tom,
> 
> interestingly, this depends on the computer from which I access the “example.html” (orcid-js is installed on our web-server).   Calling example.html from within my institute produces much fewer errors, than accessing it from home.
> 
> As mentioned before, the code hicks-up over utf-8 codes (Umlauts, accents).  In the best case this produces a "[Error] Failed to load resource: the server responded with a status of 500 (Internal Server Error)” when accessing the “pub.orcid.org http://pub.orcid.org/” record, and the record is skipped.  If the utf-8 code appears in the publication title, the code aborts in citeproc.js: TypeError: null is not an object (evaluating 'Item[variable].length’)
> 
> If I call example.html from my home computer (outside the institute domain) the list of errors is much longer and more diverse, including errors when calling crossref, etc.  I’ll compile a list of these one of the coming evenings.
> 
> The bottom line is that for the ORCID records that I administer, the orcid-js package is not yet usable.  I am however very much interested in it and hope to be able to contribute to its improvement.
> 
> Best wishes,
> 
> Ulli
>

**Comment by ulli-steiner** [comment]
Tom,

still further to my last two emails:

Running example.html of orcid-js from home produces even more errors:

[Error] XMLHttpRequest cannot load http://data.crossref.org/10.1364%2FOE.18.004356. Request header field Accept-Encoding is not allowed by Access-Control-Allow-Headers.

My guess us that there is an "Access-Control-Allow-Origin" statement missing somewhere.

Best wishes,

Ulli

**Comment by TomDemeranville** [comment]
@ulli-steiner 

I have updated the library to call the callback as citations are resolved, which speeds up page rendering and also means errors do not cause the whole list to fail.  

I am now looking at where and why those failures are happening.  I'm also going to look at the character encoding issues. It's good to have some decent test data, thanks!

Tom.

**Comment by ulli-steiner** [comment]
Tom,

great!  Let me know whether I can help.

Best wishes,

Ulli

**Comment by TomDemeranville** [comment]
## RE: UTF-8.

a) I've stopped the failures happening in the orcid-js client when the title (or any other field) is empty.  This was happening if the title contained accented characters and there was no existing bibtex citation to use. 
b) I've also submitted a fix to the core ORCID service that will enable it to parse accented characters.  Due to the limitations of the jbibtex parser we use, I have had to replace accented characters with their non-accented equivilents e.g. é -> e.  Although not ideal, this will work and enable a citation to be created.  I'll post again when it's live.

## Other notes:

a) I've found that some citeproc returned from crossref (and possibly datacite) contains the \ufffd character.  This is the annoying � you see in the results and there's not much I can do about it this end.  example

`curl -L -H 'Accept: application/vnd.citationstyles.csl+json' http://dx.doi.org/10.1140/epje/e2004-00005-8`

Shows it in one of the contributors names.  This is most likely not the fault of crossref but the depositing client.

`{"indexed":{"date-parts":[[2015,12,26]],"date-time":"2015-12-26T07:47:30Z","timestamp":1451116050224},"reference-count":0,"publisher":"Springer Science + Business Media","issue":"3","published-print":{"date-parts":[[2003,11,1]]},"DOI":"10.1140\/epje\/e2004-00005-8","type":"journal-article","created":{"date-parts":[[2004,7,22]],"date-time":"2004-07-22T02:02:08Z","timestamp":1090461728000},"page":"375-381","source":"CrossRef","title":"Capillary instabilities by fluctuation induced forces","prefix":"http:\/\/id.crossref.org\/prefix\/10.1140","volume":"12","author":[{"affiliation":[],"family":"Morariu","given":"M. D."},{"affiliation":[],"family":"Sch\ufffdffer","given":"E."},{"affiliation":[],"family":"Steiner","given":"U."}],"member":"http:\/\/id.crossref.org\/member\/297","container-title":"The European Physical Journal E - Soft Matter","deposited":{"date-parts":[[2011,6,12]],"date-time":"2011-06-12T09:37:56Z","timestamp":1307871476000},"score":1.0,"subtitle":[],"issued":{"date-parts":[[2003,11,1]]},"alternative-id":["8VMRGX2TWMKKYMAJ"],"URL":"http:\/\/dx.doi.org\/10.1140\/epje\/e2004-00005-8","ISSN":["1292-8941","1292-895X"],"subject":["Biotechnology","Biophysics","Materials Science(all)","Chemistry(all)","Surfaces and Interfaces"]}`

b) I'll look at bad datacite request URLs tomorrow.

**Comment by ulli-steiner** [comment]
Tom,

I have tried again to run the example code:

http://www.am-institute.ch/assets/site/js/orcid/orcid-js/example.html http://www.am-institute.ch/assets/site/js/orcid/orcid-js/example.html

Interestingly, the outcome depends on the browser.  One the Mac, only Firefox renders the entire list, while Safari and Chrome render only 39 entries.  While there are no browser errors, the javascript console throw up loads of “failed to load resource” errors, both from orcid and cross-ref.

And, it is still slow ...

Best wishes,

Ulli

**Comment by ulli-steiner** [comment]
Tom,

UTF-8 b):  the ORCID web-interface and http://feed.labs.orcid-eu.org do not seem to have this problem ...

Other notes:  yes, Crossref is not very good with special characters, and the import from cross-ref to Orcid is even worse.  In the case when a  \ufffd appaears, can you simply discard this record and use the one from ORCID instead?  I have fixed all umlaut problems resulting from the Crossref import into ORCID by hand.  I hope this will one day be fixed.

Best wishes,

Ulli

**Comment by ulli-steiner** [comment]
Tom,

further to my earlier email:  the code is still quite temperamental, particularly with its browser dependence:

If I change the style to one of the nature styles, Safari and Chrome render the same number of publications as before (39), but Firefox fails with an error:  

TypeError:  Item[variable] is null, in citeproc.js  9481:1

Presumably, an empty entry in the “title” field is the culprit again.

Best wishes,

Ulli

**Comment by TomDemeranville** [comment]
@ulli-steiner Hmm.  Are you sure you've got the latest version of orcid-js?  
I am seeing 157 citations in both Firefox and Chrome for 0000-0001-5936-339X.  I expect this number to increase once the fix to the ORCID service goes live.

Time wise - it takes about 14 seconds to fetch and render all 157.  This is around 11 a second, with the first being rendered after 1.5 seconds.  I'm not sure we can make it much faster.

**Comment by TomDemeranville** [comment]
I think you're still using an old version of orcid-js. (it's still hitting the rc1 api).  I made a change to orcid-js a few minutes ago, so I would get the latests and test again.  

If you update example.html and orcid.js you should now see 204 works as citations.  Rendering starts after about 1.5 seconds and takes around 15 seconds total as the results are fetched.

Once the patch to the ORCID service goes live, I expect all works to be rendered as citations.

**Comment by ulli-steiner** [comment]
Sorry,

You are right! Something went wrong unzipping the file during the upload..

I'll try again later today!

Ulli

**Comment by ulli-steiner** [comment]
Tom,

maybe the change in ORCID have not gone live, but I am getting a rather inconsistent performance across the three browsers:
1. Safari does not render anything.  Ajax throws up an error in styles.js 1257
2. Firefox: renders one citation at a time, erasing the previous one as it goes.  This happens very slowly, with occasional internal error message on the console, when a get request fails
3. Chrome renders  45 references.  Most get requests fail with 406 and 500 errors.

Note that I modified the fetchStyleXML function in style.js:

```
        $.ajax({
                type: "GET",
                url: "http://raw.githubusercontent.com/citation-style-language/styles/master/"+styleName+".csl",
                dataType: "text",
                success: function (data) {
                    callback(data);
                },
                error: function (jqXHR, textStatus, errorThrown ){
                    console.log(jqXHR);
                }
        });
```

Best wishes,

Ulli

**Comment by TomDemeranville** [comment]
Thanks for the testing Uli.  I've no idea why you're seeing different
results to me.  It's very odd.  I'm using 0000-0001-5936-339X to test.
- Could you tell me if any of the 500 errors are for crossref and send me
  an example?
- Could you send me an example of one of the 406 errors?

**Comment by ulli-steiner** [comment]
Tom,

Here are some of the errors (there are loads of these):

[Error] Failed to load resource: the server responded with a status of 500 (Internal Server Error) (10271384, line 0)

[Error] Failed to load resource: Request header field DNT is not allowed by Access-Control-Allow-Headers. (10.1002/aenm.201501868, line 0)
[Error] XMLHttpRequest cannot load http://data.crossref.org/10.1002%2Faenm.201501868. Request header field DNT is not allowed by Access-Control-Allow-Headers.

Ulli

**Comment by ulli-steiner** [comment]
Tom,

the full console logs for three browsers are here:

https://www.dropbox.com/sh/dip20kw17lcdmz3/AADYUiy7m0i_K9PYV6QFECMla?dl=0

Also, I apologise for some of the confusing earlier emails.  It seems that Chrome caches javascipts, which explains some of browser differences I have been getting.  Now, I get a very similar behaviour from Safari and Chrome, but there are differences to Firefox, which is not surprising since the first two use Webkit and the latter Gecko as rendering engines.  The bottom line is however that for me neither work, at the moment.

Also, further to yesterdays e-mail, the failure in styles.js was my fault (typo in the raw.githubusercontent.com url).  The errors I get are after this has been fixed.

Best wishes,

Ulli

**Comment by ulli-steiner** [comment]
Tom,

one more observation:

One thing that appears to have fallen by the wayside is the numbering of the list (which worked previously).

the list is now rendered as:

1.
Scherer, M. R. J. & Steiner, U. in Electrochromic Materials and Devices (eds. Mortimer, R. J., Rosseinsky, D. R. & Monk, P. M. S.) 311–336 (Wiley-VCH, 2015). at http://www.wiley-vch.de/publish/dt/books/ISBN3-527-33610-9/

1.
Russell, T., Lin, Z., Kerle, T., Leach, K., Lin, Y., Shaeffer, E. & Steiner, U. Directing self-assembly of polymers with electric fields. Abstracts of Papers of the American Chemical Society 224, U366 (2002).

1.
Scheffold, F., Eiser, E., Steiner, U., Budkowski, A., Magerle, R., Klein, J., Fetters, L. & Lohse, D. Segment-segment interactions and miscibility in polyolefin mixtures via direct binodal determination: Comparison with SANS study. Abstracts of Papers of the American Chemical Society 215, U294 (1998).

1.
Russell, T., Thurn-Albrecht, T., Lin, Z., Kerle, T., Xu, T., Tuominen, M., Schaeffer, E. & Steiner, U. Patterning polymers from the nanoscopic to microscopic size scales. Abstracts of Papers of the American Chemical Society 221, U384–U385 (2001).

1.
Scheffold, F., Budkowski, A., Eiser, E., Steiner, U., Klein, J. & Fetters, L. Interfacial segregation from tunable polyolefinic mixtures. Abstracts of Papers of the American Chemical Society 212, 252–PMSE (1996).

1.
Steiner, U. in Nanolithography and patterning techniques in microelectonics (ed. Bucknall, D.) (CRC Press, 2005). at http://www.crcpress.com/product/isbn/9780849334474

etc.

Best wishes,

Ulli

> Tom,
> 
> the full console logs for three browsers are here:
> 
> https://www.dropbox.com/sh/dip20kw17lcdmz3/AADYUiy7m0i_K9PYV6QFECMla?dl=0 https://www.dropbox.com/sh/dip20kw17lcdmz3/AADYUiy7m0i_K9PYV6QFECMla?dl=0
> 
> Also, I apologise for some of the confusing earlier emails.  It seems that Chrome caches javascipts, which explains some of browser differences I have been getting.  Now, I get a very similar behaviour from Safari and Chrome, but there are differences to Firefox, which is not surprising since the first two use Webkit and the latter Gecko as rendering engines.  The bottom line is however that for me neither work, at the moment.
> 
> Also, further to yesterdays e-mail, the failure in styles.js was my fault (typo in the raw.githubusercontent.com http://raw.githubusercontent.com/ url).  The errors I get are after this has been fixed.
> 
> Best wishes,
> 
> Ulli
>

**Comment by TomDemeranville** [comment]
Hi @ulli-steiner 

I'm still looking at a work around for this.  It (at least in part) is happening because the javascript ajax request 'forgets' the http headers when following a 303 redirect.  This means that the request to crossref is incomplete and fails with a 406 not acceptable error.  Unfortunately, 303 redirects are completely transparent to Javascript and cannot be modified.  The webkit bug for this is still open: https://bugs.webkit.org/show_bug.cgi?id=104551 

Firfox had a similar issue but it was fixed in 2014.  I think.  https://bugzilla.mozilla.org/show_bug.cgi?id=401564

**Comment by ulli-steiner** [comment]
Tom,

If I understand this correctly, you first acquire a record from ORCID, and then use the DOI to check on crossref whether it is up-to-date?  If so, can you not simply discard the unsuccessful Crossref query and display the Orcid record instead?

Best wishes,

Ulli

**Comment by TomDemeranville** [comment]
This problem is that ORCID does not contain enough bibliographic
information to create a proper citation (unless it contains Bibtex).  So we
attempt to get the full citation metadata from the DOI which could be
crossref, but could also be datacite or another DOI registration agency.  I
am going to modify the code to go to crossref directly, skipping doi.org.
If that fails I will try datacite.  If that fails I will use ORCID.

Tom.

**Comment by ulli-steiner** [comment]
Tom,

then I don’t understand why Webkit browsers render only ca. 40 citations.  All my ORCID entries have a Bibtex field - if the import does not include one, I add it by hand (this is why the ability to upload Bibtex into ORCID was so important).  If your code does what you write below, it should render all records, even if crossref and datacite fail.  Also,   http://feed.labs.orcid-eu.org only renders records with Bibtex entries, and it does a good job with my ORCID ID.  

My guess is that there probably is another hick-up in the code, preventing the fall-back to ORCID

Ulli

**Comment by TomDemeranville** [comment]
It doesn't fall back to ORCID at the moment if datacite or crossref fail.  They're failing at the mo due to bugs but should never fail in production.

I've just patched orcid-js to not use doi.org.  It now goes directly to crossref and if that fails, goes to datacite.  I've tested with your profile in Safari and apart from one work failing at ORCID, it's working for me.  

FYI - the numbering is a product of the Citation style you're using.  Some include a number.  If you fetch them one at a time, the number is always 1.  Not much I can do about that if you want to render as they're fetched.  If you change the parameter in example.html to false, it will fetch them all in one go and get the numbers right, but it will not update until ALL metadata has been retrieved.  Try a different style is my advice.

**Comment by ulli-steiner** [comment]
Tom,

Great!  Now it works!  The record that fails to resolve is one “aded by hand”, which does not have a doi or isbn number.  Concerning the styles, no worries, I will probably anyway write my own.

I think orcid-js has now reached a level of maturity that allows me to use it.  I am currently quite busy, but I will try to implement this over the next few weeks, to replace http://feed.labs.orcid-eu.org.

One more question/request. Can the build-up of the list be done asynchronously, so that the records appear more quickly, as the individual queries are completed>  There is still a bit of a wait until the list is rendered (to be fair, the wait is probably comparable to that of http://feed.labs.orcid-eu.org).

Many thanks for writing this.

Best wishes,

Ulli

**Comment by ulli-steiner** [comment]
Tom,

how do I access the retrieved metadata?

Ulli

**Comment by ulli-steiner** [comment]
Dear Tom,

I am in the process of transitioning from using the old feed to your code.

It all works well, with one exception:

For one of my records the lookup fails:

Failed to load resource: the server responded with a status of 500 (Internal Server Error)"

{"response-code":500,"developer-message":"org.jbibtex.TokenMgrError: Lexical error at line 1, column 49.  Encountered: \"\u00b5\" (181), after : \"\"","user-message":"Something went wrong in ORCID.","error-code":9008,"more-info":"http://members.orcid.org/api/api-error-codes”}

It fails looking up the following URL:  https://pub.orcid.org/v2.0_rc2/0000-0001-5936-339X/work/13315885?_=1469015411486

The URL: https://pub.orcid.org/v2.0_rc2/0000-0001-5936-339X/work/13315885 however produces a well formatted record.

I am wondering whether you can help me to resolve this.

With best wishes,

Ulli
